tests/blobstore_common_tests.py

```diff
@@ -48,6 +48,17 @@ def test_get_content_type(self):
                 self.test_fixtures_bucket,
                 "test_good_source_data_DOES_NOT_EXIST")
 
+    def test_get_last_modified_date(self):
+        last_modified = self.handle.get_last_modified_date(
+            self.test_fixtures_bucket,
+            "test_good_source_data/0")
+        self.assertTrue(isinstance(last_modified, date))
```

Comment:
get_last_modified_date returns `datetime`.  why are you asserting it's a `date`?